ia/src/zappyIa.py

```
@@ -1,22 +1,49 @@
-        if sys.argv[1] == "-help":
-            info.info()
-        sys.stderr.write("Invalid number of arguments (-help for more informations).\n")
-        sys.exit(84)
+def decode_review(text):
+    return ' '.join([reverse_word_index.get(i, '?') for i in text])
 
-    args = process.iaProcess(sys.argv)
+# print(len(test_data[0]), len(test_data[1]))
 
-    game.game(args)
+#model down here
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)